### DISCUSSION TITLE
Reduce the amount of DATA of an imported CTF

### DISCUSSION THREAD

**Discussion by bibiboom89** [mtex-toolbox/mtex#1998] [Ask Anything] [answered]
Hi,

I got a ctf file containing almost 4 millions pixels.
This is way too heavy for my computer and it prevents me to do any further work on the data.
I would like to know if there is a way to downsize these data by a factor F.
Like for example doing grid a 2x2 or 3x3 pixels and averaging the orientations (which would reduce the amount of data by 4 or 9) ?

Thank you in advance for your help,
Regards,
Remy

**Comment by zmichels** [comment, marked as answer]
https://mtex-toolbox.github.io/EBSD.reduce.html

**Reply by bibiboom89** [reply]
oh the function exist. I didn't know that 'reduce' was the right term for that.
Thank you a lot and have a great weekend :)
